app/models/author.rb

```diff
@@ -0,0 +1,18 @@
+class Author
+  include ActiveModel::Validations
+  attr_accessor :first_name, :last_name, :email
+
+  def initialize(params = {})
+    @first_name = params[:first_name]
+    @last_name = params[:last_name]
+    @email = params[:email]
+  end
+
+  validates :first_name, presence: true, length: { minimum: 2 }
+  validates :last_name, presence: true, length: { minimum: 2 }
+  validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'invalid email format' }
+
+  def to_s
```

Comment:
no test for this method

Reply:
I think you missed it in `spec/models/author_spec.rb`. It is the first test case on that file.

Reply:
> no test for this method

I think you missed it in spec/models/author_spec.rb. It is the first test case on that file.

Reply:
Correct, sorry. But I just realized that you mixed two separate tasks.